### DISCUSSION TITLE
Future of Teams Toolkit

### DISCUSSION THREAD

**Discussion by peterjbardenhagen** [microsoft/teams-sdk#2248] [Q&A]
With the future of agentic AI and Micrsoft recently presenting M365 Agents that includes publishing to Teams, which is the best platform to produce chatbots on going forward. Is Teams AI / Teams Toolkit future proofed or will it be replaced?

**Comment by corinagum** [comment]
Teams AI and Teams Toolkit are two different teams - but neither are going away!

In terms of developing bots with AI, we continue to recommend Teams AI SDK for Teams-oriented bots. Agent SDK is currently not at language or functional parity for Teams. 

Over time, Teams AI SDK will be updating our dependencies to use Agent SDK instead of Bot Framework. That work will start with the C# library. 

For Teams Toolkit, the extension and cli are great ways to streamline moving your app from local development to production, and none of those hoops will be going away any time soon, so it will continue to be an important tool for building Teams bots.
